src/devicecheck.cs

```diff
@@ -44,4 +47,30 @@ interface DCDevice {
 		[Export ("generateTokenWithCompletionHandler:")]
 		void GenerateToken (DCDeviceGenerateTokenCompletionHandler completion);
 	}
+
+	[NoWatch, NoTV, NoMac]
+	[iOS (14,0)]
+	[DisableDefaultCtor]
+	[BaseType (typeof (NSObject))]
+	interface DCAppAttestService {
+
+		[Static]
+		[Export ("sharedService")]
+		DCAppAttestService SharedService { get; }
+
+		[Export ("supported")]
+		bool Supported { [Bind ("isSupported")] get; }
+
+		[Async]
+		[Export ("generateKeyWithCompletionHandler:")]
+		void GenerateKey (Action<NSString, NSError> completionHandler);
```

Comment:
Any particular reason for NSString here?

Reply:
no, missed it :)